### ISSUE TITLE
Enhance some networks to set `bias=False` for conv layers

### ISSUE THREAD

**Issue by yiheng-wang-nv** [Project-MONAI/MONAI#2768]
If a conv layer is directly followed by a batch norm layer, the bias in the conv layer is not need, because the `subtract the mean` manipulation will cancel out the effect of bias.

[Reference:](https://pytorch.org/tutorials/recipes/recipes/tuning_guide.html)

In MONAI's `Convolution` class, the default value of `bias` is `True`, and the following networks/blocks cannot configure the `bias` parameter when using `Convolution`:

- `SimpleASPP`
- `HighResNet`
- `VNet`
- `UNet`
- `BasicUNet`
- `AutoEncoder`

**Comment by yiheng-wang-nv** [comment]
Therefore, shall we add `bias` parameters for these networks/blocks?

Or shall we add a judgement as follow in all related palces?
```
if norm == "batch" and adn_ordering[0] == "N":
conv_params["bias"] = False

conv_block = Convolution(**conv_params)
```

@rijobro @Nic-Ma @wyli @ericspod
